blackbox/docs/sql/system.txt

```diff
@@ -728,9 +728,10 @@ The table schema is as follows:
 |     Column Name      |           Description            | Return Type |
 +======================+==================================+=============+
 | ``schema_name``      | The schema name.                 | ``String``  |
-|                      | This will be "blob" for          |             |
-|                      | shards of blob tables and "doc"  |             |
-|                      | for shards of common tables.     |             |
+|                      |                                  |             |
+|                      | This will be "blob" for shards   |             |
+|                      | of blob tables and "doc" for     |             |
```

Comment:
thinks this should be corrected to something like: "... and "doc" for shards of common tables without a defined schema"